validator_client/src/cli.rs

```diff
@@ -1,509 +1,498 @@
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("http-allow-keystore-export")
-                .long("http-allow-keystore-export")
-                .requires("http")
-                .help("If present, allow access to the DELETE /lighthouse/keystores HTTP \
-                    API method, which allows exporting keystores and passwords to HTTP API \
-                    consumers who have access to the API token. This method is useful for \
-                    exporting validators, however it should be used with caution since it \
-                    exposes private key data to authorized users.")
-                .action(ArgAction::SetTrue)
-                .help_heading(FLAG_HEADER)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("http-store-passwords-in-secrets-dir")
-                .long("http-store-passwords-in-secrets-dir")
-                .requires("http")
-                .help("If present, any validators created via the HTTP will have keystore \
-                    passwords stored in the secrets-dir rather than the validator \
-                    definitions file.")
-                .action(ArgAction::SetTrue)
-                .help_heading(FLAG_HEADER)
-                .display_order(0)
-        )
-        /* Prometheus metrics HTTP server related arguments */
-        .arg(
-            Arg::new("metrics")
-                .long("metrics")
-                .help("Enable the Prometheus metrics HTTP server. Disabled by default.")
-                .action(ArgAction::SetTrue)
-                .help_heading(FLAG_HEADER)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("metrics-address")
-                .long("metrics-address")
-                .requires("metrics")
-                .value_name("ADDRESS")
-                .help("Set the listen address for the Prometheus metrics HTTP server.")
-                .default_value_if("metrics", ArgPredicate::IsPresent, "127.0.0.1")
-                .action(ArgAction::Set)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("metrics-port")
-                .long("metrics-port")
-                .requires("metrics")
-                .value_name("PORT")
-                .help("Set the listen TCP port for the Prometheus metrics HTTP server.")
-                .default_value_if("metrics", ArgPredicate::IsPresent, "5064")
-                .action(ArgAction::Set)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("metrics-allow-origin")
-                .long("metrics-allow-origin")
-                .requires("metrics")
-                .value_name("ORIGIN")
-                .help("Set the value of the Access-Control-Allow-Origin response HTTP header. \
+    styles = get_color_style(),
+    next_line_help = true,
+    term_width = 80,
+    disable_help_flag = true,
+    disable_help_subcommand = true,
```

Comment:
I understand we are matching the syntax from the `DatabaseManager` but just curious what the reason is for disabling the in-built help flag?

Also if we do disable it, we'll need our own version like in the `DatabaseManager`, see `database_manager/src/cli.rs:66`

Reply:
we're disabling the built in help flag because we want it to be displayed under the Flags section via 
  `.help_heading(FLAG_HEADER)`. By default the help flag isdisplayed under Options, which is where we list cli params that take an argument. Also we need `display_order(0)` so that its displayed in alphabetical order in the Flags section

Reply:
i've redefined the help flag as global in the parent lighthouse cli flag definition and was able to get rid of this here